src/wordpress-plugins/yoast-seo.php

```diff
@@ -193,7 +193,10 @@ private function reset_tracking() {
 	private function reset_configuration_wizard() {
 		\update_user_meta( \get_current_user_id(), 'wpseo-dismiss-configuration-notice', 'no' );
```

Comment:
I think this user meta is removed in the upgrade routine for 14.4 added in https://github.com/Yoast/wordpress-seo/pull/14917, so I guess this line should be removed?